parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -15,10 +15,7 @@ disableSshd() {
   systemctl_stop 20 5 10 sshd || exit 3
   retrycmd 120 5 25 systemctl disable sshd || exit 3
 }
-disableTimeSyncd() {
```

Comment:
Why are we removing step to disable time synced?

Reply:
systemd-timesyncd is not loaded in the latest base image.

We get the following error 
==> azure-arm: + for i in $(seq 1 $retries)
==> azure-arm: + timeout 10 systemctl daemon-reload
==> azure-arm: + timeout 10 systemctl stop systemd-timesyncd
==> azure-arm: Failed to stop systemd-timesyncd.service: Unit systemd-timesyncd.service not loaded.

https://msazure.visualstudio.com/One/_build/results?buildId=85324592&view=logs&j=963c4a5b-e1d2-573c-c288-d285fd86ae62&t=1ab4c3c4-6b77-5dcb-105f-0d715d69d005